Cesium3DTilesSelection/test/TestTilesetSelectionAlgorithm.cpp

```diff
@@ -563,31 +563,32 @@ TEST_CASE("Test additive refinement") {
   SECTION("Load external tilesets") {
     ViewState viewState = zoomToTileset(tileset);
 
-    // 1st frame. Root will get rendered first and 5 of its children are loading
-    // since they meet sse
+    // 1st frame. Root, its child, and its four grandchildren will all be
+    // rendered because they meet SSE, even though they're not loaded yet.
```

Comment:
I see "render" getting used even when tiles aren't loaded in yet -- would it be more accurate to say tiles get "selected" vs. "rendered"?

Though, it seems like this verbiage is built into the API already (`tilesToRenderThisFrame`), so maybe it's better to stick with "render". But I hope our documentation clarifies that `render ~= selection`, and `render != content show up on screen` 🙂

Reply:
Yeah this is a good point. It's a bit of a quirk of this whole system. Arguably, tiles that are not renderable should never be selected/rendered, in general. That would make sense, and be least likely to surprise people. My thought process when I went the other way was that - just maybe! - it could be useful to know which tiles _would be_ rendered if they were loaded. For example, maybe you could render a grey cloudy thing in the space of their bounding box, and that would be better than a hole.

So whether this is a good idea or not, Cesium Native currently refers to selection as "rendering", and the rendered tiles aren't necessarily actually renderable. Changing this might be an excellent idea, but I think it's outside the scope of this PR.

Reply:
> But I hope our documentation clarifies that render ~= selection, and render != content show up on screen

I think it probably makes that apparent, but doesn't say so explicitly.